### PR TITLE
Add a note for 0.5.0 release.

### PR DESCRIPTION
**Pull request by kgolab**
@bskiba , please have a look.

### CODE REVIEW

File: vertical-pod-autoscaler/README.md [hunk 1]
```diff
@@ -35,11 +35,21 @@ procedure described below.
...
+
+### Notice on removal of v1beta1 version (>=0.5.0)
+
+**NOTE:** In 0.5.0 we disabled the old version of the API - `autoscaling.k8s.io/v1beta1`.
+The VPA objects in this version will no longer receive recommendations and
+existing recommendations will be removed. The objects will remain present though
+and a ConfigUnsupported condition will be set on them.
+
+To receive recommendations please migrate to `autoscaling.k8s.io/v1beta2` API,
+as described below in the next section.
```
Comment (bskiba) [inline vertical-pod-autoscaler/README.md]: nit, I'd say either below or in the next section :)

File: vertical-pod-autoscaler/README.md [hunk 2]
```diff
@@ -35,11 +35,21 @@ procedure described below.
...
+suggest using the [latest 0.3 version](https://github.com/kubernetes/autoscaler/blob/vpa-release-0.3/vertical-pod-autoscaler/README.md).
+
+### Notice on removal of v1beta1 version (>=0.5.0)
+
+**NOTE:** In 0.5.0 we disabled the old version of the API - `autoscaling.k8s.io/v1beta1`.
+The VPA objects in this version will no longer receive recommendations and
+existing recommendations will be removed. The objects will remain present though
+and a ConfigUnsupported condition will be set on them.
+
+To receive recommendations please migrate to `autoscaling.k8s.io/v1beta2` API,
```
Comment (bskiba) [inline vertical-pod-autoscaler/README.md]: Also we might want to mention that the reasons for API change are listed there.

File: vertical-pod-autoscaler/README.md (hunk 1 above)
Comment (kgolab) [reply vertical-pod-autoscaler/README.md]: Done.

File: vertical-pod-autoscaler/README.md (hunk 2 above)
Comment (kgolab) [reply vertical-pod-autoscaler/README.md]: Done.

**Comment by bskiba** [comment]
Thanks. Do you think we should wait for the release to be cut before posting this?

**Comment by kgolab** [comment]
I think so - otherwise we might temporarily confuse someone.

**Comment by bskiba** [comment]
Cool putting on hold for now

**Pull request merged**